pkg/reading/service_read_issue.go

```diff
@@ -1,25 +1,31 @@
 package reading
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/configuration"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/helpers"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/http/rest"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/result"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/service"
 	"net/http"
+	"os"
 )
 
 type ServiceReadIssue struct {
-	issueId   string
-	parentKey string
-	fieldKey  string
-	fieldName string
+	issueId     string
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [struct field issueId should be issueID](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)